changelog/2657.feature.rst

```diff
@@ -0,0 +1,3 @@
+Added ionization energy data from NIST to the `~plasmapy.particles.Particle` class.
+This can now be accessed using the `~plasmapy.particles.Particle.ionization_energy` attribute
+from the `~plasmapy.particles.Particle` class.
```

Comment:
```suggestion
Added ionization energy data from NIST to the |Particle| class.
This can now be accessed using the `~plasmapy.particles.particle_class.Particle.ionization_energy` attribute
from the |Particle| class.
```
📚 We've defined some reStructuredText substitutions (`docs/_global_substitutions.py`) for things like `Particle` which we can use here.  For PlasmaPy's docs, we also have to put the full namespace of the object, including the module name (for better or for worse).